gulpfile.ts

```diff
@@ -1,7 +1,7 @@
 import gulp from 'gulp';
 import tasks from './tasks';
 
-gulp.task('db.dropTables', tasks.db.dropTables) ;
-gulp.task('db.init', tasks.db.init);
-gulp.task('db.restart', tasks.db.restart);
-gulp.task('db.truncate', tasks.db.truncate);
+gulp.task('db.dropTables', () => tasks.db.dropTables()) ;
+gulp.task('db.init', () => tasks.db.init());
+gulp.task('db.restart', () => tasks.db.restart());
+gulp.task('db.truncate', () => tasks.db.truncate());
```

Comment:
Do these changes make a difference? Intuitively I'd say this works the same as before.

Reply:
@moshababo has done these.